back/app/services/permissions/phase_permissions_service.rb

```diff
@@ -173,12 +174,26 @@ def attending_event_denied_reason_for_action
     # Helper methods
 
     def posting_limit_reached?
-      return false if phase.pmethod.supports_multiple_posts?
-      return true if phase.ideas.published.exists?(author: user)
+      allow_posting_again_after = phase.pmethod.allow_posting_again_after || 10.years
+      return false if allow_posting_again_after == 0.seconds
```

Comment:
Would it make sense to also add a guard clause `return true if !allow_posting_again_after`?

Reply:
This would not work as we still need to check if ideas have been posted